runtime/src/lib.rs

```diff
@@ -568,10 +618,6 @@ pub type Executive = frame_executive::Executive<
 	(),
 >;
 
-#[cfg(feature = "runtime-benchmarks")]
-#[macro_use]
-extern crate frame_benchmarking;
-
```

Comment:
Why?

Reply:
Unintended commit. Thanks for pointing this.